src/utils/createCompositeLogger.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function createCompositeLogger (primaryLogger, additionalLoggers) {
+  var loggers = [primaryLogger].concat(additionalLoggers)
+  return function compositeLogger (name, level, date, components) {
+    for (var i = 0; i < loggers.length; i++) loggers[i](name, level, date, components)
```

Comment:
Is there any reason you're using `for` loops in this PR instead of slapdash's `_.each` ?